src/sx/src/StyleCollectorAtNode.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import { type PrintableNode, type PrintConfig } from './StyleCollectorNode';
+
+/**
+ * Represents any "at" rule:
+ *
+ * ```
+ * '@media screen': {
+ *   'fontSize': 14,
+ *   ':hover': {
+ *     color: 'pink',
+ *   },
+ *   '@media (max-width: 12cm)': {
+ *     color: 'blue',
+ *   },
+ * },
+ * ```
+ *
+ *   ↓ ↓ ↓
+ *
+ * ```
+ * {
+ *   name: '@media screen',
+ *   nodes: [
+ *     StyleCollectorNode,
+ *     StyleCollectorPseudoNode,
+ *     StyleCollectorAtNode,
+ *   ],
+ * }
+ * ```
+ */
+export default class StyleCollectorAtNode implements PrintableNode {
+  atRuleName: string;
+  nodes: Set<PrintableNode>;
+
+  constructor(atRuleName: string, nodes: Set<PrintableNode>) {
+    this.atRuleName = atRuleName;
+    this.nodes = nodes;
+  }
+
+  // eslint-disable-next-line no-unused-vars
+  print(config?: PrintConfig): string {
```

Comment:
Will this config be used later?

Reply:
It won't, but there is an interface `PrintableNode` so Flow requires it. However, Eslint complains it's unused. It seems like I have to suppress one of them. 🤔